### ISSUE TITLE
Client is not mining

### ISSUE THREAD

**Issue by phobo5-it** [Bytom/bytom#898]
#### Environment

- OS: Windows Server 2012 R2 Standard

#### Problem Description

After 4 days of client in sync and mining set to On (with 4 CPU working of 12 available) it seems that anything is mined (anything in transaction/balance nor looking for my address in web explorers)

**Comment by hybridtangent** [comment]
Yes, I'm seeing the same problem. I haven't hit a block and I've been mining since mainnet was released. Please fix! Based on difficulty we should be see at least a few blocks.
